WordPress/src/main/java/org/wordpress/android/util/config/JetpackInstallFullPluginFeatureConfig.kt

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.annotation.Feature
+import org.wordpress.android.util.config.JetpackInstallFullPluginFeatureConfig.Companion.JETPACK_INSTALL_FULL_PLUGIN_REMOTE_FIELD
+import javax.inject.Inject
+
+@Feature(JETPACK_INSTALL_FULL_PLUGIN_REMOTE_FIELD, true)
```

Comment:
**Question:** Is the default value supposed to be `true` here? This would have the effect that partial changes behind this flag will show up to the user as soon as they hit the production build, is that what we expect here? I see the original issue here #17831 says it should default to `false`, so I am just commenting to double-check.

**Question:** Also, somewhat related to the point above, shouldn't the `@FeatureInDevelopment` annotation be used while the feature is in development, which would avoid showing the feature (and committed PRs with partial changes) in production builds even if the remote field (`jetpack_install_full_plugin_remote_field`) is turned on? 

Then as the last step after pushing the last commit that makes the feature "complete", we replace the `FeatureInDevelopment` with `Feature` annotation + remote field specification. Does that make sense?

Reply:
> Is the default value supposed to be true here?

Thanks for catching that, @thomashorta ! That was a copy + paste that I forgot to change.

> shouldn't the @FeatureInDevelopment annotation be used while the feature is in development,

Makes sense. I'm going to push this change too.